kola/tests/misc/network.go

```diff
@@ -207,11 +213,17 @@ func NetworkInitramfsSecondBoot(c cluster.TestCluster) {
 
 	// verify that the network service was started
 	found := false
+	// In v249, some services description have been updated.
+	// More details: https://github.com/systemd/systemd/commit/4fd3fc66396026f81fd5b27746f2faf8a9a7b9ee
+	// In v252, the printed line stoped using a description of the unit, going to the unit name.
```

Comment:
If we don't do so already we should also make sure to use `status-unit-format=combined` (can also be set as `StatusUnitFormat=combined` in `system.conf`, best through a drop in under `/usr`, I guess)

Reply:
Sure, can be done, but do you mean here to do it via a userdata in the test case or just in general, as a part of our production image? If the former, then I don't know how is this supposed to work, since putting a config snippet during ignition run probably won't do much, since systemd is already running. If the latter, then I'd do it as a follow up. Which will allow us to maybe simplify some tests in the long run, but not any time soon, as other channels still have a different configuration.

Reply:
In general, I think this makes sense but we would have to update the strings here again ;)

Reply:
Added it to my TODO list.

Reply:
The meson option `status-unit-format-default` can be set to `combined`, this is a bit better than doing it via a drop-in.

Reply:
But if it's worth to minimize the ebuild changes, this here in the `init` repo should be good:
```
diff --git a/systemd/system.conf.d/10-status-combined.conf b/systemd/system.conf.d/10-status-combined.conf
new file mode 100644
index 0000000..64200c4
--- /dev/null
+++ b/systemd/system.conf.d/10-status-combined.conf
@@ -0,0 +1,2 @@
+[Manager]
+StatusUnitFormat=combined
```

Reply:
Yeah, I was thinking about using a drop-in too.